subt/zmq-subt-x4.json

```diff
@@ -118,6 +126,13 @@
 
               ["rosmsg.gas_detected", "detector.gas_detected"],
 
+              ["app.pose2d", "lora.pose2d"],
+              ["lora.cmd", "app.cmd"],
+              ["reporter.artf_all", "lora.artf"],
+              ["lora.artf_xyz", "reporter.artf_xyz"],
+              ["lora.raw", "rosmsg.broadcast"],
+              ["rosmsg.radio", "lora.radio"],
```

Comment:
Ok. I have no idea how this works.

Reply:
The first two lines are copy & paste from System - i.e. report regularly position and accept commands (like PAUSE, return home etc. - not used at the moment).
The 3rd line is sending all collected artifacts instead just found artifact.
4th line forwards received artifacts from other robots
5th line sends LoRA output to broadcast to all robots
6th line forwards received radio data to LoRA
does this help?

Reply:
Partially. I am currently stuck at thinking about the relation of `lora` _driver_ for LoRa device from Frantisek and `CommsClient`.